integration-tests/spring-data-jpa/src/main/resources/import.sql

```diff
@@ -77,9 +77,10 @@ INSERT INTO unit(id, name) VALUES (1, 'Delivery Unit');
 INSERT INTO unit(id, name) VALUES (2, 'Sales and Marketing Unit');
 INSERT INTO team(id, name, unit_id) VALUES (10, 'Development Team', 1);
 INSERT INTO team(id, name, unit_id) VALUES (11, 'Sales Team', 2);
-INSERT INTO employee(id, user_id, first_name, last_name, team_id) VALUES (100, 'johdoe', 'John', 'Doe', 10);
-INSERT INTO employee(id, user_id, first_name, last_name, team_id) VALUES (101, 'petdig', 'Peter', 'Digger', 10);
-INSERT INTO employee(id, user_id, first_name, last_name, team_id) VALUES (102, 'stesmi', 'Stella', 'Smith', 11);
+INSERT INTO employee(id, user_id, first_name, last_name, team_id, email) VALUES (100, 'johdoe', 'John', 'Doe', 10, 'johdoe@acme.com');
+INSERT INTO employee(id, user_id, first_name, last_name, team_id, email) VALUES (101, 'petdig', 'Peter', 'Digger', 10, 'petdig@acme.com');
+INSERT INTO employee(id, user_id, first_name, last_name, team_id, email) VALUES (102, 'stesmi', 'Stella', 'Smith', 11, 'stesmi@acme.com');
+INSERT INTO employee(id, user_id, first_name, last_name, team_id, email) VALUES (100, 'jd', 'John', 'Doe', 12, 'jd@acme.com');
```

Comment:
There is a mixup in ids here. `id` 100 is already taken and `team_id` 12 doesn't exist.